d = {"Name": "John", "Surname": "Smith"}
print(d["Smith"])
#give reason why it give key error
#Hint: A KeyError always means there is no such given key in the dictionary
